src/tagify.js

```diff
@@ -772,12 +773,12 @@ Tagify.prototype = {
      * @return {Boolean/String}  ["true" if validation has passed, String for a fail]
      */
     validateTag( tagData ){
-        var value = tagData.value.trim(),
-            _s = this.settings,
+        var _s = this.settings,
+            value = _s.preserveWhiteSpace ? tagData.value : tagData.value.trim(),
             result = true;
 
         // check for empty value
-        if( !value )
+        if(_s.preserveWhiteSpace && !value || !_s.preserveWhiteSpace && !value.trim() )
```

Comment:
`!value.trim()` is enough here, it shouldn't matter about the `preserveWhiteSpace` setting, but only make sure there is any value at all and not multiple spaces for example